### PR TITLE
Update publisher

### PR DESCRIPTION
**Pull request by urso**
Update publisher to use recent publisher refactoring in libbeat.

Depends on elastic/libbeat#69

### CODE REVIEW

**Comment by tsg** [comment]
LGTM, requires merging though.

File: packetbeat.go [hunk 1]
```diff
@@ -166,20 +166,22 @@ func (pb *Packetbeat) Setup(b *beat.Beat) error {
 
 	pb.over = make(chan bool)
 
-	logp.Debug("main", "Initializing filters")
-	afterInputsQueue, err := filters.FiltersRun(
-		config.ConfigSingleton.Filter,
-		EnabledFilterPlugins,
-		b.Events,
-		b.Stop)
+	/*
```
Comment (ruflin) [inline packetbeat.go]: I would suggest to remove this commented out code.

File: tests/test_0016_nop_filter.py [hunk 2]
```diff
@@ -1,7 +1,10 @@
+import unittest
 from pbtests.packetbeat import TestCase
 
 
 class Test(TestCase):
+    @unittest.skip(
+        "Disabled until filters have been added to processing chain again")
```
Comment (ruflin) [inline tests/test_0016_nop_filter.py]: Can you open an issue for this in case it is not done directly? Otherwise we probably forget about it.

**Comment by ruflin** [comment]
+1

File: tests/test_0016_nop_filter.py (hunk 2 above)
Comment (urso) [inline tests/test_0016_nop_filter.py]: See elastic/libbeat#94

File: packetbeat.go (hunk 1 above)
Comment (urso) [inline packetbeat.go]: would like to keep it in for reference until elastic/libbeat#94 is done

File: protos/http/http.go [hunk 3]
```diff
@@ -181,8 +182,12 @@ func (http *Http) GetPorts() []int {
 	return http.Ports
 }
 
-func (http *Http) Init(test_mode bool, results chan common.MapStr) error {
+const (
```
Comment (andrewkroh) [inline protos/http/http.go]: Is there a merge mistake? I think these are not used.

Comment (urso) [inline protos/http/http.go]: oh, nice one. will check. thanks

Comment (urso) [inline protos/http/http.go]: done

**Pull request merged**